### DISCUSSION TITLE
Files on microSD card not being deleted

### DISCUSSION THREAD

**Discussion by edmond0167** [mcguirepr89/BirdNET-Pi#385] [General]
Hi Everyone,
I've had great success running BirdNET-Pi on a Pi4.
Twice now the microSD card has filled up completely and the system becomes unresponsive.
I manually used the available tools to removal all data. This worked.
However within one day the microSD card filled up completely.
I think all my settings are correct because I ticked to manage files (delete if storage space fills).
I've completed all recent updates.
Has anyone experienced this issue?

Log attached below.

Thank
<img width="932" alt="Screen Shot 2022-06-17 at 7 49 03 am" src="https://user-images.githubusercontent.com/61327779/174179608-9a4ede1f-2fad-4616-8f1b-f04a6536c136.png">
s

**Comment by ehpersonal38** [comment]
Hi there, sorry to hear you're having this issue. If you could remove enough space for commands to work (but we still want total used disk space to be greater than 95%) and then paste the output of the following command:
`bash $HOME/BirdNET-Pi/scripts/disk_check.sh`

**Reply by edmond0167** [reply]
Hi Had to remove power and reboot to gain control of the Pi.
Ran the command (thanks).
Output in Terminal below over two screen captures
<img width="925" alt="Screen Shot 2022-06-17 at 8 45 28 am" src="https://user-images.githubusercontent.com/61327779/174190625-89807987-a35b-4f66-a922-f380ebaac491.png">
<img width="920" alt="Screen Shot 2022-06-17 at 8 45 39 am" src="https://user-images.githubusercontent.com/61327779/174190648-eaf69f1f-4d6b-48e1-abbf-d5743f9b476b.png">
.

**Reply by edmond0167** [reply]
df -h shows 100% disk utilisation. I don't think I can enter the app at the moment because even my Browser on the Pi is refusing to open
<img width="427" alt="Screen Shot 2022-06-17 at 8 52 28 am" src="https://user-images.githubusercontent.com/61327779/174191230-be528811-a2e0-4c75-8468-dbe9b018a140.png">
.

**Reply by ehpersonal38** [reply]
First thing I notice is that that disk_check is very old, at least a month or more. It's using a command that was removed. So I would start with getting updated, maybe remove a lot of files and try 
```
sudo -E chown -R $USER:$USER ~/Bird*
bash $HOME/BirdNET-Pi/scripts/update_birdnet.sh`
```

I'll be able to help more tomorrow

**Reply by edmond0167** [reply]
Thanks - managed to free up some disk space by deleting directories.
<img width="507" alt="Screen Shot 2022-06-17 at 9 15 24 am" src="https://user-images.githubusercontent.com/61327779/174193368-147c7fc2-c56a-4ff2-b440-7d7c454a5770.png">

**Reply by edmond0167** [reply]
Tried to run both commands you posted.
First one was ok.
Had issues with second command.
terminal output below
<img width="723" alt="Screen Shot 2022-06-17 at 9 16 21 am" src="https://user-images.githubusercontent.com/61327779/174193511-8cec9696-a315-4ee2-9489-b65e50ffb837.png">
w

**Comment by edmond0167** [comment]
Looks like there is an error with the updates.
I also tried with the BirdNET portal.
Just wondering if I need to change my directory structure.
Thanks for all your help. I think we're almost there
<img width="935" alt="Screen Shot 2022-06-17 at 9 23 05 am" src="https://user-images.githubusercontent.com/61327779/174194174-849a1593-ecd1-4707-89e6-658006c6021d.png">
.

**Reply by ehpersonal38** [reply]
This looks really messed up; I think it could be related to your user not being "pi," but I'm far from certain on that. Going to have to defer to @mcguirepr89 on this one. 🐤

**Reply by edmond0167** [reply]
I don't mind doing a complete re-install and set the Pi back to user pi.
I can do that later today. It seems worth trying.

**Reply by ehpersonal38** [reply]
Yeah that'll certainly fix your issue. Be sure to consult [this wiki section](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Backup-and-Restore-the-Database) for backing up and restoring your DB, if you'd like to keep all your data.

And enjoy the updates to your system! A lot has changed in the past 2 months. Be sure to set up your Flickr API key in Tools -> Settings if you'd like bird images displayed next to each detection.

**Reply by edmond0167** [reply]
Thanks for all your help. I'll do this today and check that it works. 
BirdNET is great because we plan on using it to monitor revegetation efforts. We need to increase numbers of small woodland birds and decrease numbers of Noisy Minors. This is a great tool for students to use.

**Reply by edmond0167** [reply]
I think I found the issue.
Did the re-install as user "pi" but could still not do upgrades.
I'd been using the curl function on the Core Electronic web site.
curl -sL https://coreelec.io/birdnet | bash
But I still got the same error and was unable to update.

Then I tried the curl function from the GitHub site (https://github.com/mcguirepr89/BirdNET-Pi)
And it seems to have worked.
curl -s https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/newinstaller.sh | bash

Very happy for your assistance. 
I'll monitor things and let you know.

**Reply by ehpersonal38** [reply]
That's great! Sorry to hear you're having trouble with their repo. Enjoy your BirdNET-Pi! 🐤
